app/sass/settings/_liberator.scss

```diff
@@ -18,3 +19,6 @@ $inuit-widths-columns: (
 
 $base-font-stack: Georgia, Cambria, "Times New Roman", Times, serif;
 
+$inuit-enable-margins--large: true;
+$inuit-enable-responsive-margins: true;
+$inuit-generate-spacing-alias: s;
```

Comment:
Not sure why this is here... I'm not 100% sure but I think `$inuit-generate-spacing-alias` is internal variable not meant to be used outside it's module.